python-sdk/src/langwatch/model_providers.py

```
@@ -0,0 +1,117 @@
+
+def _raise_for_status(response: httpx.Response, *, operation: str = "") -> None:
+    """Map HTTP error status codes to appropriate exceptions."""
+    if response.is_success:
+        return
+
+    status = response.status_code
+    detail = ""
+    try:
+        body = response.json()
+        detail = body.get("message") or body.get("error") or ""
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)